app/index.js

```diff
@@ -482,7 +482,7 @@ app.on('ready', () => {
         title: 'Save password?',
         message: message,
         buttons: ['Yes', 'No'],
-        defaultId: 0,
+        defaultId: 1,
```

Comment:
@replacepreg your comment was removed. Feel free to try again without profanity if you'd like to have a productive conversation.